### DISCUSSION TITLE
Elasticsearch status error in security onion

### DISCUSSION THREAD

**Discussion by SomeiahCS** [Security-Onion-Solutions/securityonion#8887] [Unsupported Versions] [answered]
Hi,
I am using Security onion v2.3.110 using security onion official image for AWS. The instance type is c5.2xlarge.

![image](https://user-images.githubusercontent.com/115447536/194806113-e9ca25b2-ba04-4d5f-b6e9-b4b690cb80e3.png)

After running sudo salt-call state.highstate we get below error

![image](https://user-images.githubusercontent.com/115447536/194806310-0271a0d4-eae0-4d6c-908b-c81f1e4e52b3.png)

The SOC Grid page shows fault status.

Getting the below error from docker logs `docker logs so-elasticsearch`
![image](https://user-images.githubusercontent.com/115447536/194807344-a0d763b1-8224-4b86-bdcc-91570aafbaaa.png)

Below is the VM settings info

![image](https://user-images.githubusercontent.com/115447536/194808301-d670e388-900f-4b0b-b177-cd702bf0b4e6.png)

while checking the docker logs for so-elasticsearch it uses approx 5Gb for committing reserved memory using Native Memory Allocation. but in the VM settings it shows Max heap size is 3.32 G.

Kindly help us to fix this!!!

**Comment by dougburks** [comment]
Have you modified the Elasticsearch heap size?
https://docs.securityonion.net/en/2.3/elasticsearch.html#heap-size

**Reply by SomeiahCS** [reply]
Hi @dougburks 
Thanks for quick response, I tried changing the esheap up to 6373m (vs the ~5.5GB default for my system) and tried restarting the so-elasticsearch, but it takes lot of time to restart and doesn't producing any results and it keeps on loading.

elasticsearch:
  mainip: 'xxx.xx.xx.xx'
  mainint: 'eth0'
  esheap: '6373m'
  esclustername: '{{ grains.host }}'
  node_type: ''
  es_port: 9200
  log_size_limit: 100
  node_route_type: 'hot'

**Reply by dougburks** [reply]
> while checking the docker logs for so-elasticsearch it uses approx 5Gb for committing reserved memory using Native Memory Allocation. but in the VM settings it shows Max heap size is 3.32 G.

How much RAM does your instance have?

What is the output of `free`?

> I am using Security onion v2.3.110 

2.3.110 is a few months old:
https://blog.securityonion.net/2022/03/security-onion-23110-now-available.html

Have you tried the latest version?

**Reply by Arun28** [reply]
Hi @dougburks 

I am a colleague of Someiah. We are using 16GB RAM and 1TB SSD with instance type: c5.2xlarge

**Reply by dougburks** [reply]
What is the output of `free`?

> I am using Security onion v2.3.110

2.3.110 is a few months old:
https://blog.securityonion.net/2022/03/security-onion-23110-now-available.html

Have you tried the latest version?

**Reply by Arun28** [reply]
We are using only 26%

![image](https://user-images.githubusercontent.com/20242939/196129130-5b62b022-6f44-4716-b339-4c74ecb876d8.png)

**Reply by dougburks** [reply]
`df` shows disk usage, but that's not the same as `free` which shows memory usage. So I'll ask for the third time:

What is the output of `free`?

> I am using Security onion v2.3.110

2.3.110 is a few months old:
https://blog.securityonion.net/2022/03/security-onion-23110-now-available.html

Have you tried the latest version?

**Reply by SomeiahCS** [reply]
Hi @dougburks please find the output of `free`
![image](https://user-images.githubusercontent.com/115447536/196271158-1dc9ff09-df8f-4d90-91f4-fea1ccbc2594.png)

**Reply by dougburks** [reply]
Looks like you still have some free memory. Have you tried increasing your Elasticsearch heap size?

And I'll ask for the fourth time: 2.3.110 is a few months old, have you tried the latest version?

**Comment by Arun28** [comment, marked as answer]
Previously we have used only 16GB Memory and we have extended to 32 GB Memory in AWS instance. Now Elasticsearch is stable and we can able to increase the heap size. Thanks for your time, this issue is solved.
